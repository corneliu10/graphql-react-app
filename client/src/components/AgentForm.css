.formContainer {
  margin: 2rem 20rem;
  border: 1px solid black;
}
