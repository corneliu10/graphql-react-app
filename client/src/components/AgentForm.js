import React from "react";
import Card from "@material-ui/core/Card";
import CardContent from "@material-ui/core/CardContent";
import CreateAgent from "./CreateAgent";

import "./AgentForm.css";

function AgentForm() {
  return (
    <Card className="formContainer">
      <CardContent>
        <CreateAgent />
      </CardContent>
    </Card>
  );
}

export default AgentForm;
