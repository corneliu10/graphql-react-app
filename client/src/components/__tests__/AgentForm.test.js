// Auto-generated do not edit

/* eslint-disable import/no-extraneous-dependencies */
/* eslint-disable no-undef */
import React from "react";
import renderer from "react-test-renderer";
import AgentForm from "../AgentForm";
import { ApolloClient } from "apollo-client";
import { InMemoryCache } from "apollo-cache-inmemory";
import { ApolloProvider } from "react-apollo";
import { createUploadLink } from "apollo-upload-client";

const link = createUploadLink({
  uri: "http://localhost:4000"
});

const client = new ApolloClient({
  link,
  cache: new InMemoryCache()
});

describe("AgentForm test", () => {
  it("AgentForm should match snapshot", () => {
    const component = renderer.create(
      <ApolloProvider client={client}>
        <AgentForm />
      </ApolloProvider>
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
