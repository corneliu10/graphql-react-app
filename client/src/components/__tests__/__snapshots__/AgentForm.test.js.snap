// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AgentForm test AgentForm should match snapshot 1`] = `
<div
  className="MuiPaper-root MuiCard-root formContainer MuiPaper-elevation1 MuiPaper-rounded"
>
  <div
    className="MuiCardContent-root"
  >
    <div
      className="container"
    >
      <form
        action="#"
        onReset={[Function]}
        onSubmit={[Function]}
      >
        <div>
          <div
            className="MuiFormControl-root MuiTextField-root field"
          >
            <label
              className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
              data-shrink={false}
            >
              Email
            </label>
            <div
              className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
              onClick={[Function]}
            >
              <input
                aria-invalid={false}
                autoFocus={false}
                className="MuiInputBase-input MuiInput-input"
                disabled={false}
                name="email"
                onAnimationStart={[Function]}
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                placeholder="Email"
                required={false}
                type="email"
                value=""
              />
            </div>
          </div>
        </div>
        <div>
          <div
            className="MuiFormControl-root MuiTextField-root field"
          >
            <label
              className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
              data-shrink={false}
            >
              Name
            </label>
            <div
              className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
              onClick={[Function]}
            >
              <input
                aria-invalid={false}
                autoFocus={false}
                className="MuiInputBase-input MuiInput-input"
                disabled={false}
                name="name"
                onAnimationStart={[Function]}
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                placeholder="Name"
                required={false}
                type="text"
                value=""
              />
            </div>
          </div>
        </div>
        <div>
          <div
            className="MuiFormControl-root MuiTextField-root field"
          >
            <label
              className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
              data-shrink={false}
            >
              Phone Number
            </label>
            <div
              className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
              onClick={[Function]}
            >
              <input
                aria-invalid={false}
                autoFocus={false}
                className="MuiInputBase-input MuiInput-input"
                disabled={false}
                name="phone"
                onAnimationStart={[Function]}
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                placeholder="Phone Number"
                required={false}
                type="text"
                value=""
              />
            </div>
          </div>
        </div>
        <div>
          <div
            className="MuiFormControl-root MuiTextField-root field"
          >
            <label
              className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
              data-shrink={false}
            >
              Address
            </label>
            <div
              className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
              onClick={[Function]}
            >
              <input
                aria-invalid={false}
                autoFocus={false}
                className="MuiInputBase-input MuiInput-input"
                disabled={false}
                name="address"
                onAnimationStart={[Function]}
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                placeholder="Address"
                required={false}
                type="text"
                value=""
              />
            </div>
          </div>
        </div>
        <div>
          <div
            className="MuiFormControl-root MuiTextField-root field"
          >
            <label
              className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
              data-shrink={false}
            >
              Zip Code
            </label>
            <div
              className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
              onClick={[Function]}
            >
              <input
                aria-invalid={false}
                autoFocus={false}
                className="MuiInputBase-input MuiInput-input"
                disabled={false}
                name="zipCode"
                onAnimationStart={[Function]}
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                placeholder="Zip Code"
                required={false}
                type="text"
                value=""
              />
            </div>
          </div>
        </div>
        <section
          className="dropzone"
        >
          <div
            onBlur={[Function]}
            onClick={[Function]}
            onDragEnter={[Function]}
            onDragLeave={[Function]}
            onDragOver={[Function]}
            onDrop={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            tabIndex={0}
          >
            <input
              autoComplete="off"
              multiple={true}
              onChange={[Function]}
              onClick={[Function]}
              style={
                Object {
                  "display": "none",
                }
              }
              tabIndex={-1}
              type="file"
            />
            <p>
              Drag 'n' drop some files here, or click to select files
            </p>
          </div>
        </section>
        <button
          className="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={0}
          type="submit"
        >
          <span
            className="MuiButton-label"
          >
            Submit
          </span>
        </button>
      </form>
    </div>
  </div>
</div>
`;
